### ISSUE TITLE
Corruption when building XCI

### ISSUE THREAD

**Issue by alligatorshoes** [julesontheroad/NSC_BUILDER#120]
Not really sure where to start here, but when I try to convert a new NSP (Star Wars: Jedi Knight Academy) to XCI, I'm getting corruption when verifying the output XCI.

NSP file verification:

```***************
DECRYPTION TEST
***************
01008CA00FAE8000 - Content.CONTROL
	2962e337d7b52fda8642c18e2b38d60d.nca	  -> is CORRECT
01008CA00FAE8001 - Content.CONTROL
	3fa7013e47733441911fde7dda45e6d0.nca	  -> is CORRECT
01008CA00FAE8000 - Content.PROGRAM
	47fe29d705240a5f929010a63e7c9c2f.nca	  -> is CORRECT
01008CA00FAE8001 - Content.PROGRAM
	7eb2751dbb53cf481b28285754cb2f3e.nca	  -> is CORRECT
01008CA00FAE8000 - Content.META
	9b19add0f8bdc3e761581ddf2d50f1af.cnmt.nca -> is CORRECT
01008CA00FAE8000 - Content.MANUAL
	cdf55e735a5b3eeccc9ebda4e16a7436.nca	  -> is CORRECT
Content.TICKET
	01008ca00fae8000000000000000000b.tik	  -> is CORRECT

VERDICT: NSP FILE IS CORRECT

****************
SIGNATURE 1 TEST
****************
01008CA00FAE8000 - Content.CONTROL
    > 2962e337d7b52fda8642c18e2b38d60d.nca	  -> is PROPER

01008CA00FAE8001 - Content.CONTROL
    > 3fa7013e47733441911fde7dda45e6d0.nca	  -> is PROPER

01008CA00FAE8000 - Content.PROGRAM
    > 47fe29d705240a5f929010a63e7c9c2f.nca	  -> is PROPER

01008CA00FAE8001 - Content.PROGRAM
    > 7eb2751dbb53cf481b28285754cb2f3e.nca	  -> is PROPER

01008CA00FAE8000 - Content.MANUAL
    > cdf55e735a5b3eeccc9ebda4e16a7436.nca	  -> is PROPER

01008CA00FAE8000 - Content.META
    > 9b19add0f8bdc3e761581ddf2d50f1af.cnmt.nca   -> is PROPER

VERDICT: NSP  FILE IS SAFE

***************
HASH TEST
***************
01008CA00FAE8000 - Content.CONTROL
  - File name: 2962e337d7b52fda8642c18e2b38d60d.nca
  - SHA256: 2962e337d7b52fda8642c18e2b38d60dc26b90b0adf1fe64d85aae47ef0ec36f
   > FILE IS CORRECT

01008CA00FAE8001 - Content.CONTROL
  - File name: 3fa7013e47733441911fde7dda45e6d0.nca
  - SHA256: 3fa7013e47733441911fde7dda45e6d018a964a253e2ee289a4808f93a8fe411
   > FILE IS CORRECT

01008CA00FAE8000 - Content.PROGRAM
  - File name: 47fe29d705240a5f929010a63e7c9c2f.nca
  - SHA256: 47fe29d705240a5f929010a63e7c9c2f9ed3c7c885f21b0d240909f86c009f80
   > FILE IS CORRECT

01008CA00FAE8001 - Content.PROGRAM
  - File name: 7eb2751dbb53cf481b28285754cb2f3e.nca
  - SHA256: 7eb2751dbb53cf481b28285754cb2f3e0970fd71bae155780f1fca1aca4b6f46
   > FILE IS CORRECT

01008CA00FAE8000 - Content.META
  - File name: 9b19add0f8bdc3e761581ddf2d50f1af.cnmt.nca
  - SHA256: 9b19add0f8bdc3e761581ddf2d50f1af8b3acd5b64f5d2fa3c953c6667980ac6
   > FILE IS CORRECT

01008CA00FAE8000 - Content.MANUAL
  - File name: cdf55e735a5b3eeccc9ebda4e16a7436.nca
  - SHA256: cdf55e735a5b3eeccc9ebda4e16a74366fe13a01339e0cce85796eb526e3b8db
   > FILE IS CORRECT

VERDICT: NSP FILE IS CORRECT
```
No errors when building the XCI but here's the output just in case:

```*******************************************************
Processing list [01008CA00FAE8000].txt
*******************************************************
Calculating final content:
Filename: Star Wars Jedi Knight Jedi Academy [01008CA00FAE8000] [v0].xci
        - Writing XCI header...
        - Writing XCI game info...
        - Generating padding...
        - Writing XCI certificate...
        - Writing ROOT HFS0 header...
        - Writing UPDATE partition header...
          Calculated multiplier: 1
        - Writing NORMAL partition header...
          Calculated multiplier: 1
        - Writing SECURE partition header...
          Calculated multiplier: 2

        rightsId =      b'01008ca00fae8000000000000000000b'
        titleKeyDec =   /
        masterKeyRev =  0xb

        * Appending: 47fe29d705240a5f929010a63e7c9c2f.nca
        * Appending: 2962e337d7b52fda8642c18e2b38d60d.nca
        * Appending: cdf55e735a5b3eeccc9ebda4e16a7436.nca

        rightsId =      b'01008ca00fae8000000000000000000b'
        titleKeyDec =   /
        masterKeyRev =  0xb

        * Appending: 7eb2751dbb53cf481b28285754cb2f3e.nca
        * Appending: 3fa7013e47733441911fde7dda45e6d0.nca
        * Appending: 9b19add0f8bdc3e761581ddf2d50f1af.cnmt.nca
...................................................
STILL 0 FILES TO PROCESS
...................................................
---------------------------------------------------
*********** ALL FILES WERE PROCESSED! *************
---------------------------------------------------
```

Finally, the verification of the output XCI:

```
***************
DECRYPTION TEST
***************
01008CA00FAE8000 - Content.PROGRAM
	47fe29d705240a5f929010a63e7c9c2f.nca	  -> is CORRECT
01008CA00FAE8000 - Content.CONTROL
	2962e337d7b52fda8642c18e2b38d60d.nca	  -> is CORRECT
01008CA00FAE8000 - Content.MANUAL
	cdf55e735a5b3eeccc9ebda4e16a7436.nca	  -> is CORRECT
01008CA00FAE8001 - Content.PROGRAM
	7eb2751dbb53cf481b28285754cb2f3e.nca	  -> is CORRECT
01008CA00FAE8001 - Content.CONTROL
	3fa7013e47733441911fde7dda45e6d0.nca	  -> is CORRECT
01008CA00FAE8000 - Content.META
	9b19add0f8bdc3e761581ddf2d50f1af.cnmt.nca -> is CORRECT

VERDICT: XCI FILE IS CORRECT

****************
SIGNATURE 1 TEST
****************
01008CA00FAE8000 - Content.PROGRAM
    > 47fe29d705240a5f929010a63e7c9c2f.nca	  -> is PROPER
	* TITLERIGHTS WERE REMOVED
	* Original titlerights id is : 01008CA00FAE8000000000000000000B
	* Original titlekey is       : /

01008CA00FAE8000 - Content.CONTROL
    > 2962e337d7b52fda8642c18e2b38d60d.nca	  -> is PROPER

01008CA00FAE8000 - Content.MANUAL
    > cdf55e735a5b3eeccc9ebda4e16a7436.nca	  -> is PROPER

01008CA00FAE8001 - Content.PROGRAM
    > 7eb2751dbb53cf481b28285754cb2f3e.nca	  -> was MODIFIED
	* NOT VERIFIABLE COULD'VE BEEN TAMPERED WITH

01008CA00FAE8001 - Content.CONTROL
    > 3fa7013e47733441911fde7dda45e6d0.nca	  -> is PROPER

01008CA00FAE8000 - Content.META
    > 9b19add0f8bdc3e761581ddf2d50f1af.cnmt.nca   -> is PROPER

VERDICT: XCI FILE COULD'VE BEEN TAMPERED WITH

***************
HASH TEST
***************
01008CA00FAE8000 - Content.PROGRAM
  - File name: 47fe29d705240a5f929010a63e7c9c2f.nca
  - SHA256: 23cba06dcd08a22caa711b1feec15e3fec4258bd8069ca0dff89a92dce53dc2f
  - ORIG_SHA256: 47fe29d705240a5f929010a63e7c9c2f9ed3c7c885f21b0d240909f86c009f80
   > FILE IS CORRECT

01008CA00FAE8000 - Content.CONTROL
  - File name: 2962e337d7b52fda8642c18e2b38d60d.nca
  - SHA256: 2962e337d7b52fda8642c18e2b38d60dc26b90b0adf1fe64d85aae47ef0ec36f
   > FILE IS CORRECT

01008CA00FAE8000 - Content.MANUAL
  - File name: cdf55e735a5b3eeccc9ebda4e16a7436.nca
  - SHA256: cdf55e735a5b3eeccc9ebda4e16a74366fe13a01339e0cce85796eb526e3b8db
   > FILE IS CORRECT

01008CA00FAE8001 - Content.PROGRAM
  - File name: 7eb2751dbb53cf481b28285754cb2f3e.nca
  - SHA256: 99a14f6dacd01d76a597eceb0f04d321cb245a8b53a5fd7e0d06b3d30d421259
   > FILE IS CORRUPT

01008CA00FAE8001 - Content.CONTROL
  - File name: 3fa7013e47733441911fde7dda45e6d0.nca
  - SHA256: 3fa7013e47733441911fde7dda45e6d018a964a253e2ee289a4808f93a8fe411
   > FILE IS CORRECT

01008CA00FAE8000 - Content.META
  - File name: 9b19add0f8bdc3e761581ddf2d50f1af.cnmt.nca
  - SHA256: 9b19add0f8bdc3e761581ddf2d50f1af8b3acd5b64f5d2fa3c953c6667980ac6
   > FILE IS CORRECT

VERDICT: XCI FILE IS CORRUPT
```

Any ideas?

**Comment by alligatorshoes** [comment]
Just an update, this seems to happen on any title using the following:

```masterKeyRev =  0xb```

**Comment by alligatorshoes** [comment]
Just ran into an odd situation that may help you identify the cause/problem with this, as it appears to be reproducible.

So I have my own Marvel Ultimate Alliance 3 XCI built with update 4.0 and the DLC unlocker. I recently went to rebuild it with the new 4.01 update, but ran into the above problem - all "source" files verify perfectly fine but the output XCI fails verification with corruption after building. What's interesting about this though is if I take my raw Marvel Ultimate Alliance 3 dump (version 1.0) and then add the 4.01 update and DLC unlocker, the resulting built XCI files verifies fine. It only becomes corrupt when attempting to upgrade from my older version.

**Comment by julesontheroad** [comment]
Not sure. Maybe masterkey related? I'll try to check

**Comment by alligatorshoes** [comment]
Thanks @julesontheroad. If I can help in any way or provide any more information that might help you, please let me know :smile:

**Comment by alligatorshoes** [comment]
Hi there, just wondering if there's any update on this or if there's any more information I can provide? Just tried with your latest release (0.99) and the problem is still there :(

**Comment by julesontheroad** [comment]
Hmm i forgot, i just noticed it has a secondary program as the multiprogram games do.  **01008CA00FAE8001** the corruption you see is in this one actually. Now does the file actually works? I think it may be because the verification isn't accounting something in there since it has to reacalculate back the original header. Still i'll try to check this game, sorry about forgetting about it.

**Comment by julesontheroad** [comment]
Ok, i also get that error in the verification though the file itself is fine. I need to check it out closely but I figure it's related to the ID itself probably the verification is calculating back the header's titlerights as  01008CA00FAE800**1**0000000000000000B instead of 01008CA00FAE800**0**0000000000000000B, that's can be solved easily i think I can do if the verified file is a program titleid minus last 3 characters + 000 + titlerights ending string.

**Comment by julesontheroad** [comment]
The referenced commit fixes it. I'll see if i can redo the exes tonight, I didn't realized it since multiprogram games are rare and the only once i modified are Grandia and the Hotline Miami Collection using the xci version of them, since the xci doesn't have titlerights the header restores  just changing back the isgamecard flag so i wouldn't realize with that and updates already changed the last 3 digits for 800 when restoring the header.

**Comment by alligatorshoes** [comment]
@julesontheroad Thank you so much! I can confirm that your latest commit works beautifully and it's now verifying fine. I really appreciate your hard work, thank you.

I didn't realise this was a multititle file, so the error makes sense now. I recently had most of my titles on an external USB HDD but it eventually developed filesystem corruption - that's why I run all my titles through NSC verification before knowing if they work. I'm not sure if the file worked even if the verification failed, but I'm sure it probably did.

Thanks again!

**Comment by julesontheroad** [comment]
Yeah it was working, at least on my end, modified files like converts are keygeneration patch titles are less straightforward to verify so bugs can happen when they out of scope. Files that weren't modified arereally straightforward since you can verified them directly with the public keys from Nintendo.

**Comment by doctahoo** [comment]
This happens again, this time with Super Mario 3D All-stars NSP + update v65536. Is the resulting XCI file reliable or not?

***************
DECRYPTION TEST
***************
010049900F546000 - Content.PROGRAM
	6bda1c789b220d50d1e84a4e7444483a.nca	  -> is CORRECT
010049900F546000 - Content.CONTROL
	4057dd04dfad17ac15e0d6f6855ceb58.nca	  -> is CORRECT
010049900F546000 - Content.MANUAL
	98dc89e78cfa717e1977986de3677fff.nca	  -> is CORRECT
010049900F546000 - Content.MANUAL
	8ec6ca0494636e7049c7014264a0300e.nca	  -> is CORRECT
010049900F546001 - Content.PROGRAM
	c603ea332457144e47d1d7988e4a1c21.nca	  -> is CORRECT
010049900F546001 - Content.CONTROL
	72b0cdc4fc7952c3a574e9a25f808507.nca	  -> is CORRECT
010049900F546001 - Content.MANUAL
	9a806519ba2164c4aeb433a2c4ec0bb3.nca	  -> is CORRECT
010049900F546002 - Content.PROGRAM
	a27eba379c19de9e9da9b4410fb9c330.nca	  -> is CORRECT
010049900F546002 - Content.CONTROL
	2551b21b3d5d8581255db2c221a71dc4.nca	  -> is CORRECT
010049900F546002 - Content.MANUAL
	c96cff79fb58065a4c4f75332d528336.nca	  -> is CORRECT
010049900F546003 - Content.PROGRAM
	f66a210636e595b875010cdfb2e88186.nca	  -> is CORRECT
010049900F546003 - Content.CONTROL
	a74a62e709ffd4a843e47c68051504bd.nca	  -> is CORRECT
010049900F546003 - Content.MANUAL
	78f18454b1bd169f77b7c3d4b2e1c453.nca	  -> is CORRECT
010049900F546000 - Content.META
	867596e19cf12cf6b1b1d6e7628bf9c1.cnmt.nca -> is CORRECT
010049900F546000 - Content.PROGRAM
	6c1b08e3197f55e65affb0250446ab58.nca	  -> is CORRECT
010049900F546000 - Content.CONTROL
	7ed241d1373097e00d2ab4301b599a4e.nca	  -> is CORRECT
010049900F546000 - Content.MANUAL
	0b7aa334be7215b6d7cfc5f417c9a8fa.nca	  -> is CORRECT
010049900F546000 - Content.MANUAL
	0471acf43f7faff5fc83227a8a47e5e1.nca	  -> is CORRECT
010049900F546001 - Content.PROGRAM
	09ead702b926829ffbc7da03dd238958.nca	  -> is CORRECT
010049900F546001 - Content.CONTROL
	f89220a81a3819957576d0371c9418c7.nca	  -> is CORRECT
010049900F546001 - Content.MANUAL
	9ff6fc22a9174bee8f9c5d5736ca691d.nca	  -> is CORRECT
010049900F546002 - Content.PROGRAM
	82957f8259014bbb16ceaf1ad9359136.nca	  -> is CORRECT
010049900F546002 - Content.CONTROL
	7f82f93513947adaac3e373e9eb07dd6.nca	  -> is CORRECT
010049900F546002 - Content.MANUAL
	1e10290728c5e40b06d420f814fb3a1d.nca	  -> is CORRECT
010049900F546003 - Content.PROGRAM
	8e5b28120dadbe74e54bf740606b2f95.nca	  -> is CORRECT
010049900F546003 - Content.CONTROL
	ccfe0de5e9b71b1a72e4a4f40c74a0d4.nca	  -> is CORRECT
010049900F546003 - Content.MANUAL
	255da8307abfa598f3b5c740c06e8ff2.nca	  -> is CORRECT
010049900F546800 - Content.META
	98f02e2f1b3072b42cffd0f3c0df9039.cnmt.nca -> is CORRECT

VERDICT: XCI FILE IS CORRECT

****************
SIGNATURE 1 TEST
****************
010049900F546000 - Content.PROGRAM
    > 6bda1c789b220d50d1e84a4e7444483a.nca	  -> is PROPER
	* TITLERIGHTS WERE REMOVED
	* Original titlerights id is : 010049900F546000000000000000000B
	* Original titlekey is       : A6A5141D9A1FC44F57BD6138E5D18163

010049900F546000 - Content.CONTROL
    > 4057dd04dfad17ac15e0d6f6855ceb58.nca	  -> is PROPER

010049900F546000 - Content.MANUAL
    > 98dc89e78cfa717e1977986de3677fff.nca	  -> is PROPER
	* TITLERIGHTS WERE REMOVED
	* Original titlerights id is : 010049900F546000000000000000000B
	* Original titlekey is       : A6A5141D9A1FC44F57BD6138E5D18163

010049900F546000 - Content.MANUAL
    > 8ec6ca0494636e7049c7014264a0300e.nca	  -> is PROPER

010049900F546001 - Content.PROGRAM
    > c603ea332457144e47d1d7988e4a1c21.nca	  -> is PROPER
	* TITLERIGHTS WERE REMOVED
	* Original titlerights id is : 010049900F546000000000000000000B
	* Original titlekey is       : A6A5141D9A1FC44F57BD6138E5D18163

010049900F546001 - Content.CONTROL
    > 72b0cdc4fc7952c3a574e9a25f808507.nca	  -> is PROPER

010049900F546001 - Content.MANUAL
    > 9a806519ba2164c4aeb433a2c4ec0bb3.nca	  -> was MODIFIED
	* NOT VERIFIABLE COULD'VE BEEN TAMPERED WITH

010049900F546002 - Content.PROGRAM
    > a27eba379c19de9e9da9b4410fb9c330.nca	  -> is PROPER
	* TITLERIGHTS WERE REMOVED
	* Original titlerights id is : 010049900F546000000000000000000B
	* Original titlekey is       : A6A5141D9A1FC44F57BD6138E5D18163

010049900F546002 - Content.CONTROL
    > 2551b21b3d5d8581255db2c221a71dc4.nca	  -> is PROPER

010049900F546002 - Content.MANUAL
    > c96cff79fb58065a4c4f75332d528336.nca	  -> was MODIFIED
	* NOT VERIFIABLE COULD'VE BEEN TAMPERED WITH

010049900F546003 - Content.PROGRAM
    > f66a210636e595b875010cdfb2e88186.nca	  -> is PROPER
	* TITLERIGHTS WERE REMOVED
	* Original titlerights id is : 010049900F546000000000000000000B
	* Original titlekey is       : A6A5141D9A1FC44F57BD6138E5D18163

010049900F546003 - Content.CONTROL
    > a74a62e709ffd4a843e47c68051504bd.nca	  -> is PROPER

010049900F546003 - Content.MANUAL
    > 78f18454b1bd169f77b7c3d4b2e1c453.nca	  -> was MODIFIED
	* NOT VERIFIABLE COULD'VE BEEN TAMPERED WITH

010049900F546000 - Content.PROGRAM
    > 6c1b08e3197f55e65affb0250446ab58.nca	  -> is PROPER
	* TITLERIGHTS WERE REMOVED
	* Original titlerights id is : 010049900F546800000000000000000B
	* Original titlekey is       : 9C3BB2B747D3804791492C37A706E8EC

010049900F546000 - Content.CONTROL
    > 7ed241d1373097e00d2ab4301b599a4e.nca	  -> is PROPER

010049900F546000 - Content.MANUAL
    > 0b7aa334be7215b6d7cfc5f417c9a8fa.nca	  -> is PROPER
	* TITLERIGHTS WERE REMOVED
	* Original titlerights id is : 010049900F546800000000000000000B
	* Original titlekey is       : 9C3BB2B747D3804791492C37A706E8EC

010049900F546000 - Content.MANUAL
    > 0471acf43f7faff5fc83227a8a47e5e1.nca	  -> is PROPER

010049900F546001 - Content.PROGRAM
    > 09ead702b926829ffbc7da03dd238958.nca	  -> is PROPER
	* TITLERIGHTS WERE REMOVED
	* Original titlerights id is : 010049900F546800000000000000000B
	* Original titlekey is       : 9C3BB2B747D3804791492C37A706E8EC

010049900F546001 - Content.CONTROL
    > f89220a81a3819957576d0371c9418c7.nca	  -> is PROPER

010049900F546001 - Content.MANUAL
    > 9ff6fc22a9174bee8f9c5d5736ca691d.nca	  -> is PROPER
	* TITLERIGHTS WERE REMOVED
	* Original titlerights id is : 010049900F546800000000000000000B
	* Original titlekey is       : 9C3BB2B747D3804791492C37A706E8EC

010049900F546002 - Content.PROGRAM
    > 82957f8259014bbb16ceaf1ad9359136.nca	  -> is PROPER
	* TITLERIGHTS WERE REMOVED
	* Original titlerights id is : 010049900F546800000000000000000B
	* Original titlekey is       : 9C3BB2B747D3804791492C37A706E8EC

010049900F546002 - Content.CONTROL
    > 7f82f93513947adaac3e373e9eb07dd6.nca	  -> is PROPER

010049900F546002 - Content.MANUAL
    > 1e10290728c5e40b06d420f814fb3a1d.nca	  -> is PROPER
	* TITLERIGHTS WERE REMOVED
	* Original titlerights id is : 010049900F546800000000000000000B
	* Original titlekey is       : 9C3BB2B747D3804791492C37A706E8EC

010049900F546003 - Content.PROGRAM
    > 8e5b28120dadbe74e54bf740606b2f95.nca	  -> is PROPER
	* TITLERIGHTS WERE REMOVED
	* Original titlerights id is : 010049900F546800000000000000000B
	* Original titlekey is       : 9C3BB2B747D3804791492C37A706E8EC

010049900F546003 - Content.CONTROL
    > ccfe0de5e9b71b1a72e4a4f40c74a0d4.nca	  -> is PROPER

010049900F546003 - Content.MANUAL
    > 255da8307abfa598f3b5c740c06e8ff2.nca	  -> is PROPER
	* TITLERIGHTS WERE REMOVED
	* Original titlerights id is : 010049900F546800000000000000000B
	* Original titlekey is       : 9C3BB2B747D3804791492C37A706E8EC

010049900F546000 - Content.META
    > 867596e19cf12cf6b1b1d6e7628bf9c1.cnmt.nca   -> is PROPER

010049900F546800 - Content.META
    > 98f02e2f1b3072b42cffd0f3c0df9039.cnmt.nca   -> is PROPER

VERDICT: XCI FILE COULD'VE BEEN TAMPERED WITH

***************
HASH TEST
***************
010049900F546000 - Content.PROGRAM
  - File name: 6bda1c789b220d50d1e84a4e7444483a.nca
  - SHA256: b892fcd4430db4f255ca3a1a00ef5f7d51be47b45138d6dc7923a5382667700e
  - ORIG_SHA256: 6bda1c789b220d50d1e84a4e7444483abdd2a33af7ee555fc1b22a52e64e4aff
   > FILE IS CORRECT

010049900F546000 - Content.CONTROL
  - File name: 4057dd04dfad17ac15e0d6f6855ceb58.nca
  - SHA256: 4057dd04dfad17ac15e0d6f6855ceb58d6b50b35ae6eb602b8bd7a124845b530
   > FILE IS CORRECT

010049900F546000 - Content.MANUAL
  - File name: 98dc89e78cfa717e1977986de3677fff.nca
  - SHA256: e5459f5748cfc0a350cd29b484ff02bc38d2a599458d77e2a95f0a7fc8119300
  - ORIG_SHA256: 98dc89e78cfa717e1977986de3677fffbabd57db13134dbfc3cbb386e6bc8967
   > FILE IS CORRECT

010049900F546000 - Content.MANUAL
  - File name: 8ec6ca0494636e7049c7014264a0300e.nca
  - SHA256: 8ec6ca0494636e7049c7014264a0300e693329ae67a2a558fce77d0fddfc4b87
   > FILE IS CORRECT

010049900F546001 - Content.PROGRAM
  - File name: c603ea332457144e47d1d7988e4a1c21.nca
  - SHA256: 8c05f1b1806bc68be6e24545a0cccb9940240dec70eb579d0b492ed7da9d3f3d
  - ORIG_SHA256: c603ea332457144e47d1d7988e4a1c212f06c70b71df56c36e621fbb4976b03c
   > FILE IS CORRECT

010049900F546001 - Content.CONTROL
  - File name: 72b0cdc4fc7952c3a574e9a25f808507.nca
  - SHA256: 72b0cdc4fc7952c3a574e9a25f808507db2f4fdb416cd45e7037035ad0551bd3
   > FILE IS CORRECT

010049900F546001 - Content.MANUAL
  - File name: 9a806519ba2164c4aeb433a2c4ec0bb3.nca
  - SHA256: cc7ba9b4c59fe8942770f5730a15f44368aa7f40407e31e8499e9429d0c9824b
   > FILE IS CORRUPT

010049900F546002 - Content.PROGRAM
  - File name: a27eba379c19de9e9da9b4410fb9c330.nca
  - SHA256: 74090f4c11436e5b3ccdd7f7fb549ff7fd862d33d10d4b10fba5c0eb4efdc75b
  - ORIG_SHA256: a27eba379c19de9e9da9b4410fb9c33024fe8fa04307821c624a3e0b8276f138
   > FILE IS CORRECT

010049900F546002 - Content.CONTROL
  - File name: 2551b21b3d5d8581255db2c221a71dc4.nca
  - SHA256: 2551b21b3d5d8581255db2c221a71dc47f9bd6b802d61622f7351c2082178347
   > FILE IS CORRECT

010049900F546002 - Content.MANUAL
  - File name: c96cff79fb58065a4c4f75332d528336.nca
  - SHA256: abc6d0c26322a61eb0f8128b0a0dd6478c7ef4fb6e67f27255e0f84f56bb625f
   > FILE IS CORRUPT

010049900F546003 - Content.PROGRAM
  - File name: f66a210636e595b875010cdfb2e88186.nca
  - SHA256: 36ff780e11ba3a1b42884c9b1e2d762215c4f8a126339ba99f8ebd8b502ddbea
  - ORIG_SHA256: f66a210636e595b875010cdfb2e8818616e040c963a7801a6ed96ef0b4fa02ad
   > FILE IS CORRECT

010049900F546003 - Content.CONTROL
  - File name: a74a62e709ffd4a843e47c68051504bd.nca
  - SHA256: a74a62e709ffd4a843e47c68051504bdc058aa319062cd8c1eeab5ddfc5a429e
   > FILE IS CORRECT

010049900F546003 - Content.MANUAL
  - File name: 78f18454b1bd169f77b7c3d4b2e1c453.nca
  - SHA256: 736b18e8489f17e1b600fa59fa63c1cd584b070d50c4b877c15de40b4a4aabcd
   > FILE IS CORRUPT

010049900F546000 - Content.META
  - File name: 867596e19cf12cf6b1b1d6e7628bf9c1.cnmt.nca
  - SHA256: 867596e19cf12cf6b1b1d6e7628bf9c1e6a15adf8b634cfe9952565957b731c2
   > FILE IS CORRECT

010049900F546000 - Content.PROGRAM
  - File name: 6c1b08e3197f55e65affb0250446ab58.nca
  - SHA256: 1d2cad2e01005460df581a939a991d7f1f99c5e3c97d51cc5f86c44e8eb3b366
  - ORIG_SHA256: 6c1b08e3197f55e65affb0250446ab5819690c4a9cc55e1bc0dc9d866fd42b92
   > FILE IS CORRECT

010049900F546000 - Content.CONTROL
  - File name: 7ed241d1373097e00d2ab4301b599a4e.nca
  - SHA256: 7ed241d1373097e00d2ab4301b599a4e01d2561767b5f9ebcbffd1a1b3fc61ce
   > FILE IS CORRECT

010049900F546000 - Content.MANUAL
  - File name: 0b7aa334be7215b6d7cfc5f417c9a8fa.nca
  - SHA256: d9950b6ad6e1d1d7965d3c51efd41cf69a3dd5ed74427b3cceae2ca46d8cb526
  - ORIG_SHA256: 0b7aa334be7215b6d7cfc5f417c9a8fa1f2aa70bd69f58cb5e188aa86f14926b
   > FILE IS CORRECT

010049900F546000 - Content.MANUAL
  - File name: 0471acf43f7faff5fc83227a8a47e5e1.nca
  - SHA256: 0471acf43f7faff5fc83227a8a47e5e13ba4f69613c6bdeded1305f2eb1582af
   > FILE IS CORRECT

010049900F546001 - Content.PROGRAM
  - File name: 09ead702b926829ffbc7da03dd238958.nca
  - SHA256: 85dbecd5dd9392962b310af537f7399a11ab9f3e1fbae2901d9bba390e0aeb09
  - ORIG_SHA256: 09ead702b926829ffbc7da03dd238958970ffb81993979b214d583a0a1858940
   > FILE IS CORRECT

010049900F546001 - Content.CONTROL
  - File name: f89220a81a3819957576d0371c9418c7.nca
  - SHA256: f89220a81a3819957576d0371c9418c78aa39ecb516cec7fb8ef9769a396a1bd
   > FILE IS CORRECT

010049900F546001 - Content.MANUAL
  - File name: 9ff6fc22a9174bee8f9c5d5736ca691d.nca
  - SHA256: c9420a0cd92a7639f8d05f6be5fed09c6818f9d99d4c2f18e56568463c92ffcd
  - ORIG_SHA256: 9ff6fc22a9174bee8f9c5d5736ca691d3e63ff9de19a52aced45d31ad29ff7a4
   > FILE IS CORRECT

010049900F546002 - Content.PROGRAM
  - File name: 82957f8259014bbb16ceaf1ad9359136.nca
  - SHA256: 7dad46711e83728cf67091424d9c92a908589433f13050ed4577889b47c053d3
  - ORIG_SHA256: 82957f8259014bbb16ceaf1ad9359136209b2a518d810887a9acd90a74bea2fa
   > FILE IS CORRECT

010049900F546002 - Content.CONTROL
  - File name: 7f82f93513947adaac3e373e9eb07dd6.nca
  - SHA256: 7f82f93513947adaac3e373e9eb07dd646b1beb147e2bece30ebac64c5e66775
   > FILE IS CORRECT

010049900F546002 - Content.MANUAL
  - File name: 1e10290728c5e40b06d420f814fb3a1d.nca
  - SHA256: 7ab9248eb2467f83f967cae11139fdb98a375f7d31facc0a590ed1b5f67bef61
  - ORIG_SHA256: 1e10290728c5e40b06d420f814fb3a1d334c863de454d5f64f638f50dace93be
   > FILE IS CORRECT

010049900F546003 - Content.PROGRAM
  - File name: 8e5b28120dadbe74e54bf740606b2f95.nca
  - SHA256: b10862c0b78a85af6d14ddb57682c848217f338890630b6d32f06652e80e3dd6
  - ORIG_SHA256: 8e5b28120dadbe74e54bf740606b2f95ebff838621079d9d6a2ef1bbe714ac4b
   > FILE IS CORRECT

010049900F546003 - Content.CONTROL
  - File name: ccfe0de5e9b71b1a72e4a4f40c74a0d4.nca
  - SHA256: ccfe0de5e9b71b1a72e4a4f40c74a0d4cd9cedb49101d03f9197e445e8edd6da
   > FILE IS CORRECT

010049900F546003 - Content.MANUAL
  - File name: 255da8307abfa598f3b5c740c06e8ff2.nca
  - SHA256: 4164dd08952777d38a6c6cb3f8628ac02ef225e735f2efbc7550c57f9b97071d
  - ORIG_SHA256: 255da8307abfa598f3b5c740c06e8ff225b992eb98e294c653fe98212a2723ec
   > FILE IS CORRECT

010049900F546800 - Content.META
  - File name: 98f02e2f1b3072b42cffd0f3c0df9039.cnmt.nca
  - SHA256: 98f02e2f1b3072b42cffd0f3c0df90398bc033d27c8d6f89c26680203da65e9e
   > FILE IS CORRECT

VERDICT: XCI FILE IS CORRUPT
